repmgr_version.h.in

```diff
@@ -1,7 +1,7 @@
-#define REPMGR_VERSION_DATE ""
-#define REPMGR_VERSION "5.4dev"
-#define REPMGR_VERSION_NUM 50400
-#define REPMGR_EXTENSION_VERSION "5.4"
-#define REPMGR_EXTENSION_NUM 50400
-#define REPMGR_RELEASE_DATE "2022-XX-XX"
```

Comment:
I believe the XX are needed. Check the Makefiles

Reply:
If I understand, that variable affects only what we put in https://www.repmgr.org/docs/5.3/install-requirements.html, in the "current" release.

I haven't found it in any other place, maybe we just added the "XX-XX" because we didn't know in advance when a new released would be publised?

If we keep the Xs, we would see this
```
$ awk '/^#define REPMGR_RELEASE_DATE / { print $3; }' repmgr_version.h.in | cut -d '"' -f 2
2024-XX-XX
```
I'll generate the docs to discard.